data/json/overmap/overmap_special/specials.json

```
@@ -7334,5 +7334,27 @@
     "//": "Low chance to find a matching location, so always try to place it",
     "occurrences": [ 100, 100 ],
     "flags": [ "LAB", "UNIQUE" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "o_lightindustry_scen",
+    "overmaps": [
+      { "point": [ 1, 0, 0 ], "overmap": "s_lightindustry_scen_road_0" },
```

```suggestion
      { "point": [ 1, 0, 0 ], "overmap": "s_lightindustry_scen_road_0_north" },
```
All of these need to have north at the end like this example
```  {
    "type": "overmap_special",
    "id": "motel-twd",
    "overmaps": [
      { "point": [ 0, 0, 0 ], "overmap": "motel_twd_1_north" },
      { "point": [ 1, 0, 0 ], "overmap": "motel_twd_2_north" },
      { "point": [ 0, 0, 1 ], "overmap": "motel_twd_second_floor_north" }
```